Tasks/AzureAppServiceManage/task.loc.json

```diff
@@ -54,6 +54,13 @@
       "helpMarkDown": "ms-resource:loc.input.help.WebAppName"
     },
     {
+      "name": "SpecifySlot",
```

Comment:
Keep back variable name in sync with value for name. If it is SpecifySlot keep the same thing